scripts/token-hud-extension.js

```diff
@@ -136,6 +140,49 @@ class TokenHudExtension {
 
    }
 
+   
+   static async addArmorTokenTip(app, html, data, actor) {
+
+        if (game.scenes.active.isView === false)
+        return;
+
+        const actorCharacteristics = actor.data.data.characteristics;
+        const ap = actor.prepareItems();
+        let toughness = actorCharacteristics.t.value;
+        let shieldAp = (ap.AP.shield - ap.totalShieldDamage);
+
+        let TooltipAp = game.i18n.localize('CHAR.T') + ": " + toughness +"; " + game.i18n.localize("SHEET.ShieldAP") + ": " + (ap.AP.shield - ap.totalShieldDamage) + "&#10;";
```

Comment:
Why not use `shieldAp` value (from line152) in Tooltip instead of calculating it again?